iothub/service/samples/PnpServiceSamples/noComponentSample/Program.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices;
+using Microsoft.Azure.Devices.Shared;
+
+namespace PnpServiceSample
+{
+    public class Program
+    {        
+        // Get connection string and device id inputs
+        private static readonly string hubConnectionString = Environment.GetEnvironmentVariable("IOTHUB_CONNECTION_STRING");
```

Comment:
add `s_` prefix to private static variables, here and everywhere else

Reply:
Just curious, i've seen this a lot in the samples. Why do we prefix with s_? The reason I ask is that generic C# naming convention asks us not to use underscores.

Reply:
Interesting, that is what we specify in our [coding guidelines](https://github.com/Azure/azure-iot-sdk-csharp/blob/master/doc/coding-style.md):
We use `_camelCase` for internal and private fields and use `readonly` where possible. Prefix instance fields with `_`, static fields with `s_` and thread static fields with `t_`. When used on static fields, `readonly` should come after `static` (i.e. `static readonly` not `readonly static`)